app/config.py

```diff
@@ -57,7 +57,7 @@ class Test(Config):
     DEBUG = True
 
     # used during tests as a domain name
-    SERVER_NAME = "document-download.test"
+    SERVER_NAME = "download.document-download-frontend-test"
```

Comment:
Optional: You could consider changing the `FRONTEND_HOSTNAME` to be `document-download.test` and then setting the `SERVER_NAME` to be `download.document-download.test`

The reason I suggest this is it is weird having the API route contain the word 'frontend' in. I don't think any our tests benefit from having the word frontend particularly either.

But equally, super minor, maybe not worth the effort to change for potential small benefit

Reply:
Confusingly, the value of `SERVER_NAME` is never used, but we still needed to have the variable defined otherwise there was an error. I've pushed a commit which removes it and hopefully makes things a little clearer.

Reply:
Ah but `SERVER_NAME` is implicitly used by `url_for`: https://flask.palletsprojects.com/en/2.2.x/api/#flask.Flask.url_for